docs/6.x/cluster.md

```diff
@@ -208,6 +208,41 @@ $ curl -sk https://localhost:7575 | python -m json.tool
 
 In this case the response HTTP status code will be `503 Service Unavailable`.
 
+### Cluster Status History
+
+Running `gravity status history` will display a history of changes to the 
+cluster status. 
+
+Example output may looks something like the following:
+
+```bsh
+$ gravity status history
+Feb  6 21:36:07 UTC [Node Degraded]     node=node-1
+Feb  6 21:36:11 UTC [Node Degraded]     node=node-2
+Feb  6 21:36:25 UTC [Node Degraded]     node=node-3
+Feb  6 21:36:56 UTC [Probe Succeeded]   node=node-1 checker=node-status
+Feb  6 21:36:58 UTC [Probe Succeeded]   node=node-2 checker=node-status
+Feb  6 21:36:58 UTC [Probe Succeeded]   node=node-2 checker=time-drift
+Feb  6 21:36:59 UTC [Probe Succeeded]   node=node-3 checker=node-status
+Feb  6 21:37:07 UTC [Probe Succeeded]   node=node-1 checker=kube-apiserver
+Feb  6 21:37:07 UTC [Node Recovered]    node=node-1
+Feb  6 21:37:08 UTC [Probe Succeeded]   node=node-2 checker=kube-apiserver
+Feb  6 21:37:08 UTC [Node Recovered]    node=node-2
+Feb  6 21:37:11 UTC [Probe Succeeded]   node=node-3 checker=kube-apiserver
+Feb  6 21:37:11 UTC [Node Recovered]    node=node-3
+```
+
+There are just a few event types that are currently being tracked.
+- `Node Degraded` / `Node Recovered` specifies a change in the node status. The node
+key specifies the name of the node (node-1, node-2, node-3).
+- `Probe Succeeded` / `Probe Failed` specifies a change in a probe result. The checker
+key specifies the name of the health check (time-drift, kube-apiserver).
+- A few other events will be tracked and are WIP. We plan to track cluster 
+resizing, gravity upgrades, network partitions, etc...
```

Comment:
I would suggest we add a link to create a Github Issue, if someone has a feature request.   e.g. https://github.com/gravitational/gravity/issues/new/choose

Reply:
Is it possible to forward these `gravity status history` events to centralised system or is it just meant as a real-time debugging tool?

Reply:
Right now its just a debugging tool. I don't think it would take too much effort to forward these events to a centralized logging system. I'll take a look in a later time.

Reply:
In that case, I would add a note that this has been designed as debugging tool.

Reply:
> The `gravity status history` command is an additional tool to help debug issues
with a Cluster.

 I think this portion states that the main purpose of this tool is for debugging. Should we reword this to specify this is command is only for debugging and nothing more?

Reply:
@benarent good to go?